sdk/core/ts-http-runtime/src/client/clientHelpers.ts

```diff
@@ -7,7 +7,7 @@ import { bearerTokenAuthenticationPolicy } from "../policies/bearerTokenAuthenti
 import { createDefaultHttpClient } from "../defaultHttpClient.js";
 import { createPipelineFromOptions } from "../createPipelineFromOptions.js";
 import { TokenCredential, isTokenCredential } from "../auth/tokenCredential.js";
-import { KeyCredential } from "../auth/keyCredential.js";
+import { KeyCredential, isKeyCredential } from "../auth/keyCredential.js";
```

Comment:
While we are in here, can we add `type` in front of the types for this file such as `KeyCredential`?

Reply:
Since there are so many type-only imports I will follow up this in a separate PR